osu.Game.Tests/Visual/Multiplayer/TestSceneLoungeSubScreen.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Screens;
+using osu.Framework.Testing;
+using osu.Game.Graphics.Containers;
+using osu.Game.Screens.Multi.Lounge;
+using osu.Game.Screens.Multi.Lounge.Components;
+
+namespace osu.Game.Tests.Visual.Multiplayer
+{
+    public class TestSceneLoungeSubScreen : RoomManagerTestScene
+    {
+        private LoungeSubScreen loungeScreen;
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+        }
+
+        public override void SetUpSteps()
+        {
+            base.SetUpSteps();
+
+            AddStep("clear rooms", () => RoomManager.Rooms.Clear());
```

Comment:
minor point, but

```suggestion
```

as it actually looks to fire twice:

![2020-07-09-235204_228x393_scrot](https://user-images.githubusercontent.com/20418176/87094472-3d931900-c23f-11ea-9193-6d2132ec02fa.png)
![2020-07-09-235324_491x148_scrot](https://user-images.githubusercontent.com/20418176/87094578-69160380-c23f-11ea-86b2-f2e0b95993f5.png)